gems/pending/appliance_console.rb

```diff
@@ -28,7 +28,8 @@
 include HighLine::SystemExtensions
 
 require 'i18n'
-LOCALES = File.join(File.expand_path(File.dirname(__FILE__)), "appliance_console/locales/*.yml")
+locales_dir = ENV['CONTAINER'].to_s.casecmp("true") == 0 ? "container" : "appliance"
```

Comment:
I typically also don't set envvars to true/false. I would expect `export CONTAINER=1` to also work